examples/tfengine/generated/org_foundation/cicd/main.tf

```diff
@@ -24,7 +24,7 @@
 # to install the Cloud Build app and connect your GitHub repository to your Cloud project.
 
 terraform {
-  required_version = ">=0.12, <0.14"
+  required_version = ">=0.13, <0.14"
```

Comment:
Can we make this >= 0.13? CFT has also started removing a max because breaking changes are becoming less common (at least syntactically). Let's only add a max if there are known breakages in future versions.

Reply:
Done